tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/0-get-started/TC_05-sitetype-primary.cy.js

```diff
@@ -0,0 +1,109 @@
+// <reference types="Cypress" />
+import { CheckDrawerDisabled } from '../../wp-module-support/drawer.cy';
+import { CheckCardHeadingSubheading } from '../../wp-module-support/header.cy';
+import {
+	CheckHelpPanelLinks,
+	CheckIllustrationPanel,
+	CheckInfoPanel,
+	CheckIntroPanel,
+} from '../../wp-module-support/sidebar.cy';
+
+describe( 'Get Started Site Type Primary', function () {
+	before( () => {
+		cy.visit(
+			'wp-admin/?page=nfd-onboarding#/wp-setup/step/get-started/site-primary'
+		);
+		cy.wait( 3000 );
+	} );
+
+	it( 'Check if the Suppressed Drawer does not open on clicking Toggle Button', () => {
+		CheckDrawerDisabled();
+	} );
+
+	it( 'Check if Header and Subheader shows up', () => {
+		CheckCardHeadingSubheading( true );
+	} );
+
+	it( 'Check to make sure sidebar opens, content is in place and close sidebar', () => {
+		CheckIntroPanel( '__get-started-site-secondary', 'Site Type' );
+		CheckIllustrationPanel();
+		CheckInfoPanel();
+		CheckHelpPanelLinks();
+	} );
+
+	it( 'Check different Categories exist and is selectable', () => {
+		let categoryCount = 0;
+		const className = '.nfd-card-pri-category';
+		cy.get( className ).should( 'be.visible' );
+		const arr = cy.get( className );
+		arr.each( () => {
+			cy.get( className )
+				.eq( categoryCount )
+				.click()
+				.should( 'have.class', 'chosenPrimaryCategory' );
+			categoryCount += 1;
+		} );
+	} );
+
+	it( 'Check if input box exists and is editable', () => {
+		let categoryCount = 0;
+		const className = '.nfd-card-pri-category';
+		cy.get( className ).should( 'be.visible' );
+		cy.get( '.nfd-setup-primary-custom__tellus-input' )
+			.scrollIntoView()
+			.should( 'be.visible' )
+			.type( 'Not Business' );
+		const arr = cy.get( className );
+		arr.each( () => {
+			cy.get( className )
+				.eq( categoryCount )
+				.should( 'not.have.class', 'chosenPrimaryCategory' );
+			categoryCount += 1;
+		} );
+	} );
+
+	it( 'Check selecting Category resets input box', () => {
+		cy.get( '.nfd-card-pri-category-wrapper span' ).first().click();
+		cy.get( '.nfd-setup-primary-custom__tellus-input' )
+			.scrollIntoView()
+			.should( 'be.visible' )
+			.should( 'be.empty' );
+	} );
+
+	it( 'Check existence of Need Help Tag', () => {
+		cy.get( '.nfd-card-need-help-tag' )
```

Comment:
Should we check for `exists` before we `scrollIntoView`?

Reply:
We don't really need to check every time I guess because if that doesn't exist it will fail.

Reply:
Just mentioned because the test says `Check existence of Need Help Tag` and I did not see an `exists` assertion. We should probably check for `exists` instead of waiting for it to timeout.